src/main.rs

```diff
@@ -442,3 +453,13 @@ fn parse_label(labelpair: &str) -> anyhow::Result<(String, String)> {
         _ => bail!("invalid label format `{labelpair}`. Expected `key=value`"),
     }
 }
+
+fn parse_certs_for_paths(paths: Vec<PathBuf>) -> anyhow::Result<()> {
```

Comment:
Do we load certs just to make sure that they *can* be loaded and that there are some?

Reply:
Correct, the idea is to provide feedback as early as possible when the user might be providing this flag to a non-existent certificate, or potentially badly formatted one.

Otherwise these certificates wouldn't get loaded until they try to pull down an OCI image and/or report OTEL telemetry, which could be potentially minutes to hours to days later, which is far from ideal, so rather than failing late and in an obscure way, I took the approach of failing fast and loudly on initialization.

Reply:
What do you think about renaming this to reflect that? `assert_...`  or `ensure_...` might be nice?

Take it or leave it -- otherwise I'm good on the clarifying question!